vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.cpp

```diff
@@ -65,28 +77,30 @@ void DeviceInfoHolder::InitDeviceInfoHolder(const Mso::React::IReactContext &con
               strongHolder->updateDeviceInfo();
             }
           });
-    } else if (
-        auto hwnd =
-            reinterpret_cast<HWND>(XamlUIService::GetIslandWindowHandle(deviceInfoHolder->m_context->Properties()))) {
-      deviceInfoHolder->m_wmSubscription = SubscribeToWindowMessage(
-          ReactNotificationService(context.Notifications()),
-          WM_WINDOWPOSCHANGED,
-          [weakHolder = std::weak_ptr(deviceInfoHolder)](HWND hwnd, const DesktopWindowMessage &dwm) {
-            if (auto strongHolder = weakHolder.lock()) {
-              const auto pos = reinterpret_cast<WINDOWPOS *>(dwm.LParam);
-              const auto newWidth = static_cast<float>(pos->cx);
-              const auto newHeight = static_cast<float>(pos->cy);
-              const auto changed =
-                  (strongHolder->m_windowWidth != newWidth) || (strongHolder->m_windowHeight != newHeight);
-              strongHolder->m_windowWidth = newWidth;
-              strongHolder->m_windowHeight = newHeight;
-              if (changed) {
-                strongHolder->notifyChanged();
-              }
-            }
-          });
+    } else {
+        hwnd = XamlUIService::GetIslandWindowHandle(deviceInfoHolder->m_context->Properties());
     }
   }
+
+  if (hwnd) {
+    deviceInfoHolder->m_wmSubscription = SubscribeToWindowMessage(
+        ReactNotificationService(context.Notifications()),
```

Comment:
Is anyone firing this notification?  I suspect this currently requires the application to manually notify us when the window changes size.?

Reply:
Added FowardWindowMessage so that it fires now.